node/Cargo.toml

```diff
@@ -22,32 +22,32 @@ rand = { version = "0.8.5", features = ["small_rng"] }
-sc-rpc-api = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sp-block-builder = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-substrate-frame-rpc-system = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
+pallet-transaction-payment-rpc = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.43" }
+sc-rpc-api = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.43" }
+sp-block-builder = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.43" }
+substrate-frame-rpc-system = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.43" }
 
-sc-consensus-pow = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sc-consensus-manual-seal = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sc-basic-authorship = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sc-cli = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sc-client-api = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sc-consensus = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sc-executor = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sc-network = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sc-service = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sc-telemetry = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sc-transaction-pool = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sc-transaction-pool-api = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sp-api = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sp-blockchain = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sp-consensus = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sp-consensus-pow = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sp-core = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sp-inherents = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sp-runtime = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sp-timestamp = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
-sp-transaction-pool = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.42" }
+sc-consensus-pow = {git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.43" }
```

Comment:
NIT: I noticed this before, but strange way to format (left curly no space, right with a space). Maybe a good time to get it consistent.

Reply:
```suggestion
sc-consensus-pow = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.43" }
```